src/AppInstallerCLICore/Workflows/ShowFlow.cpp

```diff
@@ -52,7 +53,16 @@ namespace AppInstaller::CLI::Workflow
         }
         if (!description.empty())
         {
-            info << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelDescription << ' ' << description << std::endl;
+            bool isMultiLine = RegexFindAndReplace(description, std::regex("\n"), "\n  ");
+            info << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelDescription;
+            if (isMultiLine)
+            {
+                info << std::endl << "  " << description << std::endl;
+            }
+            else
+            {
+                info << ' ' << description << std::endl;
+            }

```

Comment:
I think this could be abstracted to a function like `ShowMultiLineField` to avoid repeating it for each one.